controller_interface/src/controller_interface_base.cpp

```diff
@@ -205,10 +205,12 @@ const rclcpp_lifecycle::State & ControllerInterfaceBase::configure()
             : ctrl_itf_params_.controller_manager_update_rate / ticks_per_controller_per_second;
 
         RCLCPP_WARN_EXPRESSION(
-          get_node()->get_logger(), !is_frequency_achievable,
-          "The requested update rate of '%ld Hz' is not achievable with the controller manager "
-          "update rate of '%d Hz'. Setting it to the closest achievable frequency '%d Hz'.",
-          update_rate, ctrl_itf_params_.controller_manager_update_rate, achievable_hz);
+          get_node()->get_logger(), !is_frequency_achievable, "%s",
+          fmt::format(
```

Comment:
Why switch to fmt? Is there a pattern from somewhere else? As before, we have been using just plain `fprintf` as this is default in ros 2 logging?